docs/usage/request.txt

```diff
@@ -4,20 +4,33 @@ Access to HttpRequest
 RPC4Django allows RPC methods to be written in such a way that they have
 access to Django's HttpRequest_ object. This can be used
 to see the type of request, the user making the request or any specific
-headers in the request object. To use this, methods must be written such 
-that they can accept arbitrary
-keyword arguments. Although currently only the HttpRequest object is sent,
+headers in the request object. To use this, methods could be written such 
+that they can accept arbitrary keyword arguments or 
+specifing request has first argument like classic django view function.
+Although currently only the HttpRequest object is sent,
 additional keyword arguments may be sent in the future.
 
 .. _HttpRequest: http://docs.djangoproject.com/en/dev/ref/request-response/
 
 ::
 
  @rpcmethod(name='rpc4django.httprequest',signature=['string'])
+ def request(request):
+     '''
+     Illustrates access to the HttpRequest object
+     '''
+     
+     return str(request)
+
+or 
+
+::
+
+ @rpcmethod(name='rpc4django.httprequest',signature=['string'])
  def request(**kwargs):
```

Comment:
Maybe you're right, it can be confusing to provide this two examples, the explicit way is better.
But we will still support the old style using kwargs since someone may want to declare a rpcmethod using `**kwargs`
An backwards compatibility is pythonic I think